index.js

```diff
@@ -114,20 +116,28 @@ exports.getKeyIndexes = function (commandName, args, options) {
       }
     }
     break;
-  case 'zunionstore':
-  case 'zinterstore':
-    keys.push(0);
-    keyStop = Number(args[1]) + 2;
-    for (i = 2; i < keyStop; i++) {
+  case 'migrate':
+    for (i = 3; i < args.length - 1; i++) {
```

Comment:
According to the doc, the third argument could also be a key: `MIGRATE host port key|"" destination-db timeout [COPY] [REPLACE] [KEYS key [key ...]]`. Shouldn't this be:

```javascript
if (args[2] === '' || args[2] === null || typeof args[2] === 'undefined') {
  for (i = 5; i < args.length - 1; i++) {
    if (args[i].toUpperCase() === 'KEYS') {
      for (var j = i + 1; j < args.length; j++) {
        keys.push(j);
      }
      break;
    }
  }
} else {
  keys.push(2);
}
```